card/mod/05_standard/set/self/all.rb

```diff
@@ -3,7 +3,7 @@
   if task = Env.params[:task]
     if Auth.always_ok?
       case task.to_sym
-      when :clear_cache           ;  Card::Cache.reset_global; Card::ViewCache.reset
+      when :clear_cache           ;  Card::Cache.reset_all; Card::ViewCache.reset
```

Comment:
Line is too long. [81/80]<br>Do not use semicolons to terminate expressions.<br>Space found before semicolon.